### PR TITLE
Add ERC20 custom minting logic and tests

### PR DESCRIPTION
**Pull request by samwerner**
### What I did

This PR makes the following changes:
- Add custom minting logic for USDC, DAI, USDT, EURS, sEURS, sETH, sUSD, ankrETH
- Add fork tests for custom minting logic

### How to verify it

`brownie test --network mainnet-fork`

### Checklist

- [x] I have confirmed that my PR passes all linting checks
- [x] I have included test cases
- [ ] I have updated the documentation (README.md)
- [x] I have added an entry to the changelog

### CODE REVIEW

File: brownie_tokens/forked.py [hunk 1]
```diff
@@ -29,6 +29,9 @@ class MintableForkToken(Contract):
     ERC20 wrapper for forked mainnet tests that allows standardized token minting.
     """
 
+    def __init__(self, address: str):
```
Comment (iamdefinitelyahuman) [inline brownie_tokens/forked.py:32]: Why is this necessary?

File: brownie_tokens/forked.py [hunk 2]
```diff
@@ -105,7 +133,31 @@ def mint_0xfE18be6b3Bd88A2D2A7f928d00292E7a9963CfC6(
...
+    # Synth sEURS
+    target_contract = Contract.from_explorer("0xC61b352fCc311Ae6B0301459A970150005e74b3E")
+    target_contract.issue(target, amount, {"from": "0x778D2d3E3515e42573EB1e6a8d8915D4a22D9d54"})
+
+
+def mint_0x57Ab1ec28D129707052df4dF418D58a2D46d5f51(
+    token: MintableForkToken, target: str, amount: int
+) -> None:
+    # Synth sUSD
+    target_contract = Contract.from_explorer("0x6C85C5198C3CC4dB1b87Cb43b2674241a30f4845")
```
Comment (iamdefinitelyahuman) [inline brownie_tokens/forked.py:160]: Why the use of `from_explorer` here and above? Is it because of the frequenly modified proxy implementations?

File: brownie_tokens/forked.py (hunk 1 above)
Comment (samwerner) [reply brownie_tokens/forked.py]: Whoops, this should be removed

**Review by iamdefinitelyahuman — APPROVED** [review]
